src/modules/people-list/people-list-item.type.tsx

```diff
@@ -0,0 +1,44 @@
+export interface IProps {
+  /** clicked on card */
+  onUserClicked?: () => void;
+
+  /** user name */
+  name?: string | undefined;
+
+  /** user marital status */
+  maritalStatus?: string | undefined;
+
+  /** user gender */
+  gender?: string | undefined;
+
+  /** church entity name */
+  churchEntityName?: string | undefined;
+
+  /** profile pic url */
+  profilePic?: string | undefined | null;
+
+  /** user id */
+  userId?: string;
+
+  /** footer element */
+  footerElement?:
+    | React.ComponentType<any>
+    | React.ReactElement
+    | null
+    | undefined;
```

Comment:
This can simply be `React.ReactElement | null | undefined`

Reply:
Done